app/controllers/api/v1/recipes_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Api::V1::RecipesController < ApplicationController
+  def index
+    if params[:user_id].present?
+      user = User.find_by(id: params[:user_id])
+      if user
+        render json: UserRecipesSerializer.new(UserRecipesPoro.new(params[:user_id], user.recipes.all)), status: 200
```

Comment:
do we need a poro for this? Since the recipe is already structured in a way we need it, wondering if it can be used straight from the serializer

Reply:
I'm not sure I understand.  How would I use the model with the serializer? I've primarily done it with Poros.